src/template.js

```diff
@@ -305,6 +305,42 @@ export class Templates {
       resolver(templateClass);
     }
   }
+
+  /**
+   * @param {!BaseTemplate} impl
+   * @param {!JSONObject} data
+   * @private
+   */
+  render_(impl, data) {
+    const root = impl.render(data);
+    const anchors = root.getElementsByTagName('a');
+    for (let i = 0; i < anchors.length; i++) {
+      const anchor = anchors[i];
+      if (!anchor.hasAttribute('href')) {
+        // Ignore anchors without href.
+        continue;
+      }
+
+      const target = anchor.getAttribute('target');
+      let updatedTarget;
+      if (target == '_top' || target == '_blank') {
```

Comment:
Should we allow `_self` too? Or can you point me to where we define what should be sanitized?